_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,210 @@
+
+2. After the transaction was confirmed, as part of a group of transactions contained in a newly-mined block being relayed across the network.
+
+There's no need for the node to receive the transaction a second time if it still has the first copy. Compact block relay (BIP152) can eliminate this redundancy by allowing a node to receive from its peers an ordered list of what transactions are included in a new block. With this knowledge, the node can use the transactions it has already received to partly or fully reconstruct the transactions part of the block for itself.  If the node doesn't receive all the transactions it needs to fully reconstruct the block, it requests the missing transactions from its peers, and then uses them to complete the block.
+
+![Compact Blocks diagram](https://raw.githubusercontent.com/bitcoin/bips/master/bip-0152/protocol-flow.png)
+
+Compact blocks provides three very important benefits to the network:
+
+1. By reducing the amount of bandwidth used by transaction relay nodes, compact blocks help offset the expected increase in bandwidth that will occur when the segwit capacity increases are enabled by miners.  This offset should allow nodes to continue operating on the network after segwit even if they're presently near their current bandwidth caps.
+
+2. By eliminating the bandwidth spike that occurs when nodes receive a new block, compact blocks may make it easier to keep a node operating on connections with limited peak bandwidth.  For example, several users have reported that receiving a new block slows down other important activity on their network, such as video conferencing, so some of those users shut down Bitcoin Core before starting those activities.  Compact blocks may eliminate those bandwidth spikes and make running Bitcoin Core less inconvenient for those users.
+
+3. Faster propagation of blocks across the network, by dramatically reducing the amount of data that needs to be transported when a new block is found.
+
+**More information:**
+
+- [Release notes][release notes compact blocks]
+
+- [Compact Block Relay FAQ](/en/2016/06/07/compact-blocks-faq/)
+
+- [BIP152][], which describes the compact blocks protocol
+
+- [Bitcoin FIBRE](http://bitcoinfibre.org/), an open source protocol and implementation that builds upon compact blocks to minimize latency for new block announcements between peers on managed networks.  FIBRE was designed and implemented alongside compact block relay version 1 and it is being used to test improvements for a subsequent version of compact block relay.
+
+## Fee filtering
+
+For several years now, Bitcoin Core nodes have used a minimum relay fee rate to determine what unconfirmed transactions they'll process, relay, and store in their individual memory pools.  Each node gets to decide its own minimum relay fee rate, and if they receive a transaction whose fee rate is below that limit, they don't add it to their memory pools or relay it to their other peers.
```

Comment:
@sdaftuar

> The min relay fee that Bitcoin Core nodes have historically had isn't a true minimum, in that free transactions would still get relayed (though rate-limited) by default.

I've added commit 2d0d7aff07535359155fb822752377abf8d19027 to mention tx priority, which hopefully clears up this inaccuracy.

> It's only due to the mempool limiting introduced in 0.12 that causes nodes to get into a state where no new transactions are accepted below some "minimum mempool fee"

If I understand your comment correctly, I think it's only correct if we assume that every node uses the same minimum relay fee.  Prior to 0.12, I almost always customized my minrelayfee, and I believe in one of the 0.11.x releases it was recommend for low-memory users to customize their minrelayfee (and, IIRC, in another 0.11.x, the default minrelayfee was temporarily increased).  This leads me to believe that even if I'm the only weirdo who customized his minrelayfee policy that historically a significant number of nodes were connecting to peers who had different minrelayfees and so bandwidth was being wasted when a node with a higher minrelayfee setting relayed transactions to a peer with a lower minrelayfee setting.

I wrote this section with that in mind, generalizing the situation across all modern releases to help keep it both correct, simple, and short for this announcement blog post which is almost certain too long already. :-)  I understand that the fixed-size mempools introduced in 0.12.0 with their variable minrelayfees aggravated the historic problem with wasted bandwidth and helped motivate the development of `feefilter`, but I'm not sure we need to go into that level of detail (but we can, if you think it's important).

If I've misunderstood your comment (or the technology), please let me know and I can make additional edits.  Thanks!